src/ledger/LedgerManagerImpl.cpp

```diff
@@ -1244,12 +1248,15 @@ LedgerManagerImpl::logTxApplyMetrics(AbstractLedgerTxn& ltx, size_t numTxs,
     auto ledgerSeq = ltx.loadHeader().current().ledgerSeq;
     auto hitRate = mApp.getLedgerTxnRoot().getPrefetchHitRate() * 100;
 
-    CLOG(DEBUG, "Ledger") << "Ledger: " << ledgerSeq << " txs: " << numTxs
-                          << ", ops: " << numOps
-                          << ", prefetch hit rate (%): " << hitRate;
-
     // We lose a bit of precision here, as medida only accepts int64_t
     mPrefetchHitRate.set_count(std::llround(hitRate));
+
+    if (Logging::logTrace("Perf"))
+    {
+        CLOG(TRACE, "Perf")
```

Comment:
related to my question on the perf partition: TRACE doesn't look right for this. As this is done once per ledger, DEBUG seemed OK. TRACE would be "I want to see performance information at the operation level" type of thing